test/integration/protocol-profile-behavior/disableBodyPruning.test.js

```diff
@@ -132,7 +109,8 @@ describe('protocolProfileBehavior', function () {
                 sinon.assert.calledOnce(testrun.response);
                 sinon.assert.calledWith(testrun.response.getCall(0), null);
 
-                var response = rawRequest; // raw request message for this request
+                // raw request message for this request
+                var response = testrun.request.getCall(0).args[2].headers.get('raw-request');
```

Comment:
split this into response and rawResponse assignment to make it easier to understand.

Reply:
Even split lines without headers like below?
`var response = testrun.request.getCall(0).args[2].stream.toString();`